src/platform/plugins/private/links/public/components/dashboard_link/dashboard_link_component.test.tsx

```diff
@@ -66,6 +69,7 @@ describe('Dashboard link component', () => {
       <DashboardLinkComponent
         link={resolvedLink}
         layout={LINKS_VERTICAL_LAYOUT}
+        textOverflow={LINK_TEXT_OVERFLOW_ELLIPSIS}
```

Comment:
We could clean these tests up by making both `layout` and `textOverflow` optional props. Then, the default values could be enforced on the component like so:

```typescript
export const DashboardLinkComponent = ({
  link,
  layout = ,
  textOverflow,
  parentApi,
}: {
  link: ResolvedLink;
  layout?: LinksLayoutType;
  textOverflow?: LinksTextOverflowType;
  parentApi: LinksParentApi;
}) => {
   <...>
}
```

Again, technically out of scope here 😅